www/styles/Nav.module.css

```diff
@@ -35,3 +37,10 @@
 .nav Button {
   margin: 5px 10px;
 }
+
+.moveOver {
+  margin-left: 100px;
```

Comment:
I'm confused, are you doing this as padding for the child element?

Reply:
i needed the agenda to move over 
so it wasnt on the left side totally but had space to be like figma

Reply:
I think the better solution would be to place `Agenda` in its own tags and style it. Having text free floating it bad practice.